cephadm.yml

```diff
@@ -0,0 +1,149 @@
+---
+
+- name: preconfigure linodes
+  import_playbook: pre-config.yml
+
+- hosts: all
+  become: yes
+  tasks:
+  - name: fetch cephadm
+    delegate_to: localhost
+    register: cephadm
+    run_once: true
+    uri:
+      force: yes
+      mode: 0755
+      return_content: yes
+      url: https://github.com/ceph/ceph/raw/octopus/src/cephadm/cephadm
+
+  - name: distribute cephadm
+    copy:
+      dest: /root/cephadm
+      group: root
+      mode: 0755
+      owner: root
+      content: "{{ cephadm.content }}"
+
+  - name: create /etc/ceph directory
+    file:
+      path: /etc/ceph
+      state: directory
+      owner: root
+      group: root
+      mode: 0755
```

Comment:
would be great, if we could move this upstream to cephadm: 

https://github.com/ceph/ceph/blob/980c286ce9b757985846261f2c377571fb1cd59e/src/cephadm/cephadm#L2416-L2417

Reply:
You mean to create the `/etc/ceph` directory if it does not exist, yes that'd be good.